src/Nest/Aggregations/AggregationResult.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Nest
+{
+	[ExactContractJsonConverter(typeof(AggregationResultJsonConverter))]
```

Comment:
Yea, I'm having trouble coming up with a batter name for `IAggregationItem` and `IAggregationResult`.  `IAggregation` would be good, but that already exists for requests.